polkadot-parachain/src/chain_spec/glutton.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::chain_spec::Extensions;
+use cumulus_primitives_core::ParaId;
+use sc_service::ChainType;
+
+/// Specialized `ChainSpec` for the Glutton parachain runtime.
+pub type GluttonChainSpec =
+	sc_service::GenericChainSpec<glutton_runtime::GenesisConfig, Extensions>;
+
+pub fn glutton_development_config() -> GluttonChainSpec {
+	GluttonChainSpec::from_genesis(
+		// Name
+		"Glutton Development",
+		// ID
+		"glutton_dev",
+		ChainType::Local,
+		move || glutton_genesis(2005.into()),
```

Comment:
Since we plan to run many (potentially 100s) of these, it will be cumbersome to add all this code every time. Can this read the `ParaId` from the chainspec or CLI?

Reply:
Good idea. I updated the code so the `ParaId` is now being read from the CLI.